src/gui/qml/FolderDelegate.qml

```diff
@@ -231,7 +231,7 @@ Pane {
 
             Button {
                 id: addSyncButton
-                text: accountSettings.accountState.supportsSpaces ? qsTr("Add Space") : qsTr("Add Folder")
+                text: accountSettings.accountState.supportsSpaces && Theme.spacesAreCalledSpaces ? qsTr("Add Space") : qsTr("Add Folder")
```

Comment:
What about calling it the same way as the account state? `Theme.supportsSpaces`

Reply:
fixed